src/demo.json

```diff
@@ -24,6 +24,33 @@
                   }
                 }
               ],
+              "items": [
```

Comment:
These don't really add anything to the demo, and I'm fine with the demo foregoing making use of the inventory system - for now at least.